packages/core/src/Modules/Cashier/Components/on-ramp-provider-popup.jsx

```diff
@@ -102,7 +102,13 @@ const OnRampProviderPopup = ({
                 <div className='on-ramp__popup-buttons'>
                     <Button.Group>
                         <Button large onClick={() => setIsOnRampModalOpen(false)} secondary text={localize('Cancel')} />
-                        <Button large onClick={onClickDisclaimerContinue} primary text={localize('Continue')} />
+                        <Button
+                            id='gtm-onramp-provider-continue'
```

Comment:
Could we make these dynamic based on the provider's name? e.g. 

```JS
id={`gtm-onramp-provider-continue--${provider.name}`}
```

Reply:
Done, thank you! 👍